docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-milestone-triggering-proc.adoc

```diff
@@ -1,2 +1,34 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-milestone-triggering-{context}']
+= Milestone configuration and triggering
+
+Case milestones can be triggered manually using a signal in the case design, or they can be configured to start automatically when a case instance starts.
+
+Milestones are configured as `Adhoc autostart` by default in the case designer. 
```

Comment:
@mswiderski This may be a bug. When I drag and drop a new milestone node in the IT Orders sample project the autostart property is indeed false. However, if I create a new case project in Business Central and a new case definition, when I add a milestone there it is autostart true. This is in the CR1 build.

Reply:
However, I will remove reference to the asset being autostart true throughout the docs as we are getting very close to freeze.